### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by bsima** [danielsz/boot-environ#2]
Is this deprecated now that weavejester/environ#30 has been merged? FWIW, boot-environ is not yet on Clojars.

**Comment by danielsz** [comment]
That is exactly right. The code in `danielsz/boot-environ` and `boot-environ` is identical. `danielsz/boot-environ` will be deprecated as soon as `boot-environ` will be released on Clojars. James has merged it in [`environ`](https://github.com/weavejester/environ), but not yet released it. Sooner or later, you may want to let it be known that you're waiting, like arrdem did, [here](https://github.com/weavejester/environ/pull/30).

**Comment by yenda** [comment]
It is available now

**Comment by loomis** [comment]
I've updated the [boot community task wiki](https://github.com/boot-clj/boot/wiki/Community-Tasks) to indicate that the task in [weavejester/environ](https://github.com/weavejester/environ) replaces this one. Hope this is OK.

**Comment by danielsz** [comment]
Absolutely. Thank you, @loomis. I went and removed the reference to the deprecated library as well.
